Resources/views/Section/view.html.twig

```diff
@@ -38,28 +38,33 @@
                 </li>
             </ul>
             <p>
-                {{ 'section.assigned.contentcount'|trans({'%contentCount%': contentCount} ) }}
+                {{ 'section.assigned.contentcount'|transchoice(contentCount, {'%contentCount%': contentCount} ) }}
             </p>
 
             <div>
-                {{ form_start(form) }}
-                    <a href="{{ path('admin_sectionedit', {'sectionId': section.id}) }}"
-                       class="pure-button ez-button{% if not canEdit %} pure-button-disabled{% endif %}" data-icon="&#xe606;"
-                            >
-                        {{ 'section.edit'|trans }}
-                    </a>
+            {% if canEdit %}
+                <a href="{{ path('admin_sectionedit', {'sectionId': section.id}) }}"
+                   class="pure-button ez-button" data-icon="&#xe606;">
+                    {{ 'section.edit'|trans }}
+                </a>
 
+                <a href="{{ path("admin_sectiondelete", {"sectionId": section.id}) }}"
```

Comment:
that should be button to post a form, deleting something with a GET request is bad.